core/src/UserSettingsDialog.html

```diff
@@ -0,0 +1,294 @@
+                storedUserSettings;
+            } else {
+                storedUserSettings = userSettingsFromStorage;
+            }
+            openEditor([key, userSettingsData[0][key]]);
+        });
+    });
+
+    function openEditor(category, event) {
+        if (event) {
+            document.querySelectorAll('.lui-us-list .fd-nested-list__link').forEach(elem => {
+                elem.classList.remove('is-selected');
+            });
+            const link = event.target.closest('.fd-nested-list__link');
+            link.classList.add("is-selected");
+            if (window.innerWidth < CSS_BREAKPOINTS.desktopMinWidth) {
+                closeNavOnCategoryClickMobile();
+            }
+        } else {
+            document.querySelectorAll('.lui-us-list .fd-nested-list__link')[0].classList.add("is-selected");
+        }
+        userSettingsSchema = category;
+        displayEditor = true;
+    }
+
+    const updateSettingsObject = (event) => {
+        storedUserSettings = event.detail.userSettings;
+    }
+
+    function storeUserSettings() {
+        LuigiConfig.storeUserSettings(storedUserSettings, previousUserSettings).then(() => {
+            dispatch('close');
+        });
+    }
+
+    function closeNavOnCategoryClickMobile() {
+        toggleNavMobile();
+    }
+
+    function toggleNavMobile() {
+        document.querySelector('.lui-usersettings-dialog').classList.toggle('usersettings-leftNavVisible');
+    }
+    function hasOpenUIicon(node) {
+        return NavigationHelpers.isOpenUIiconName(node.icon);
+    }
+
+    export function handleKeydown(event) {
+        if (event.keyCode === KEYCODE_ESC) {
+            dispatch('close');
+        }
+    }
+</script>
+
+<style>
+    .dialog-size {
+        width: 60vw;
+        height: 60vh;
+    }
+
+    .user-settings__body {
+        position: relative;
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    .user-settings__leftnav {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 200px;
+        height: 100%;
+        z-index: 2;
+    }
+
+    .fd-side-nav {
+        height: 100%;
+        width: 100%;
+        overflow-y: scroll;
+    }
+
+    .user-settings__content {
+        position: absolute;
+        left: 201px;
+        top: 0;
+        height: 100%;
+        right: 0;
+        overflow-wrap: break-word;
+        padding: 20px;
+    }
+
+    .userSettings-lui-burger {
+        margin-right: 16px;
+        cursor: pointer;
+        display: none;
+    }
+
+    .fd-nested-list__item {
+        display: flex;
+        -webkit-box-align: center;
+        align-items: center;
+        position: relative;
+        -webkit-box-flex: 1;
+        flex: 1 1 auto;
+        max-width: 100%;
+    }
+
+    .fd-nested-list__link {
+        height: 5rem;
+    }
+
+    .fd-nested-list__icon img {
+        max-width: 18px;
+        max-height: 18px;
+    }
+
+    .us-lui-list-item-info-container {
```

Comment:
A bit out of common pattern class, can it be renamed to .lui-usersettings-list-item-info-contauiner?